static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -1084,6 +1021,12 @@ $quote-mark: 35px;
         font-style: normal;
     }
 
+    .meta__extras,
+    .meta__number:not(.u-h) + .meta__number,
```

Comment:
`.meta__number:not(.u-h)` looks a bit unsafe – we shouldn't really be targetting helpers like this. if it's hidden, does it matter if a style is applied to it?